src/sentry/stacktraces/functions.py

```diff
@@ -0,0 +1,175 @@
+# coding: utf-8
+from __future__ import absolute_import
+
+import re
+
+from sentry.stacktraces.platform import get_behavior_family_for_platform
+
+
+_windecl_hash = re.compile(r'^@?(.*?)@[0-9]+$')
+_rust_hash = re.compile(r'::h[a-z0-9]{16}$')
+_cpp_trailer_re = re.compile(r'(\bconst\b|&)$')
```

Comment:
Suggestion: Let's strip or shorten `(anonymous namespace)` and `` `anonymous namespace'``.

Reply:
We can't change that now easily since this is already part of the current grouping algorithm. We would need to change quite a bit for this.